code/index.html

```diff
@@ -1,22 +1,73 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Running News</title>
+</head>
+<body>
+
     <header>
-      <!-- Some kind of logo and navbar -->
+        <img class="logo" src="https://minglebit.com/img/portfolios/app/products/gravityrunner/gravityrunner_logo.png" alt="runnerlogo"></a>
```

Comment:
The alt text we want to write as "normal" text! :)
```suggestion
        <img class="logo" src="https://minglebit.com/img/portfolios/app/products/gravityrunner/gravityrunner_logo.png" alt="runner logo"></a>
```